buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -636,7 +636,7 @@ class BeamModulePlugin implements Plugin<Project> {
     def log4j2_version = "2.20.0"
     def nemo_version = "0.1"
     // [bomupgrader] determined by: io.grpc:grpc-netty, consistent with: google_cloud_platform_libraries_bom
-    def netty_version = "4.1.118.Final"
+    def netty_version = "4.1.110.Final"
```

Comment:
I think we should relay to GCP-BOM for lower priority vulneribility rather than overwrite  dependency versions managed by GCP-BOM as it isn't tested to work amongst gcp components.